opencti-platform/opencti-front/src/components/MarkdownField.jsx

```diff
@@ -63,7 +63,7 @@ const MarkdownField = (props) => {
         {label}
       </InputLabel>
       <ReactMde
-        value={field.value}
+        value={field.value ?? ''}
```

Comment:
Reason: `value` can be null. But an input should not be initialized with null or undefined value, instead use an empty string. By so, React can properly track changes with the input.